sdks/python/apache_beam/io/tfrecordio.py

```diff
@@ -41,23 +41,20 @@
 
 
 def _default_crc32c_fn(value):
-  """Calculates crc32c of a bytes object using either snappy or crcmod."""
+  """Calculates crc32c of a bytes object using either google-crc32c or crcmod."""
 
   if not _default_crc32c_fn.fn:
     try:
-      import snappy  # pylint: disable=import-error
-      # Support multiple versions of python-snappy:
-      # https://github.com/andrix/python-snappy/pull/53
-      if getattr(snappy, '_crc32c', None):
-        _default_crc32c_fn.fn = snappy._crc32c  # pylint: disable=protected-access
-      elif getattr(snappy, '_snappy', None):
-        _default_crc32c_fn.fn = snappy._snappy._crc32c  # pylint: disable=protected-access
-    except ImportError:
+      import google_crc32c  # pylint: disable=import-error
```

Comment:
we cannot just remove snappy. To keep back compatible, we should keep this and add the check for google_crc32c.